OSPSuite.Presentation.Importer/Views/ColumnMappingControl.cs

```diff
@@ -136,6 +139,11 @@ private void onFormatChanged(object sender, EventArgs e)
 
       public event FormatChangedHandler OnFormatChanged;
 
+      public void Rebind()
+      {
+         _gridViewBinder.Rebind();
```

Comment:
Only way at the moment to let the binder knows that the underlying list of possibility for the combo box has changed